inc/namespace.php

```diff
@@ -76,8 +72,10 @@ function load_wpseo() {
 
 	// Define a fake WP SEO Premium File value if we don't have WP SEO Premium installed. This hides some of the upsell UI.
 	if ( ! class_exists( 'WPSEO_Premium' ) ) {
-		define( 'WPSEO_PREMIUM_FILE', $wpseo_file );
-		define( 'WPSEO_PREMIUM_VERSION', 8 );
+		if ( ! defined( 'WP_CLI' ) || ! WP_CLI ) {
+			define( 'WPSEO_PREMIUM_FILE', $wpseo_file );
+			define( 'WPSEO_PREMIUM_VERSION', 8 );
```

Comment:
The _specific_ version value doesn't matter, particularly. But the `WPSEO_PREMIUM_VERSION` constant is used internally for toggling some features. It seemed nicer to pass an actual version number rather than just a `true` value. This is really more territory of #58 and not specifically related to this PR.